Assets/MixedRealityToolkit.SDK/Experimental/FollowSolver/Follow.cs

```diff
@@ -371,6 +401,26 @@ private float AngleBetweenVectorAndPlane(Vector3 vec, Vector3 normal)
             return 90 - (Mathf.Acos(Vector3.Dot(vec, normal)) * Mathf.Rad2Deg);
         }
 
+        private float AngleBetweenOnXYPlane(Vector3 from, Vector3 to)
+        {
+            float angle = Mathf.Atan2(to.y, to.x) - Mathf.Atan2(from.y, from.x);
+            return SimplifyAngle(angle) * Mathf.Rad2Deg;
+        }
+
+        float AngleBetweenOnXZPlane(Vector3 from, Vector3 to)
+        {
+            float angle = Mathf.Atan2(to.z, to.x) - Mathf.Atan2(from.z, from.x);
+            return SimplifyAngle(angle) * Mathf.Rad2Deg;
+        }
```

Comment:
You can use `AngleBetweenOnPlane` and give the plane normal argument. `Vector3.forward` for xy-plane and `Vector3.up` for xz-plane.

Reply:
These functions can be removed